### PR TITLE
Remove warning about memory usage when using multiple flutter engines

### PR DESCRIPTION
**Pull request by mkustermann**

### CODE REVIEW

**Comment by mkustermann** [comment]
Isolate groups have been fully supported in JIT mode for a while now and memory overhead per additional isolate should not be meaningfully larger than in AOT.

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Review by domesticmouse — APPROVED** [review]

**Comment by sfshaza2** [comment]
I'm waiting for your lgtm, @gaaclarke.

File: src/development/add-to-app/multiple-flutters.md [hunk 1]
```diff
@@ -32,12 +32,6 @@ memory cost (~180kB) for adding additional Flutter instances. This fixed cost
 reduction allows the multiple Flutter instance pattern to be used more liberally
 in your add-to-app integration.
 
-{{site.alert.warning}}
-  Memory usage is only fully optimized in AOT mode (in profile and release
-  builds). Some memory redundancy will still be present in JIT mode (in debug
-  builds). This is tracked in [issue 74520][].
```
Comment (gaaclarke) [inline src/development/add-to-app/multiple-flutters.md:38]: Line 100 should be removed too for the link, otherwise LGTM.

Comment (mkustermann) [reply src/development/add-to-app/multiple-flutters.md:38]: Done.

**Comment by mkustermann** [comment]
Thanks for the review!

**Pull request merged**